doc/whats_new/v0.20.rst

```diff
@@ -125,6 +125,13 @@ Classifiers and regressors
   only require X to be an object with finite length or shape.
   :issue:`9832` by :user:`Vrishank Bhardwaj <vrishank97>`.
 
+Cluster
+
+- :class:`cluster.KMeans`, :class:`cluster.MiniBatchKMeans` and
```

Comment:
I think you should say that with algorithm='full', row-major ordering will be forced, improving runtime.